front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { compact } from 'lodash';
+
+import type {
+  PathPropertiesFormatted,
+  SimulationResponseSuccess,
+} from 'applications/operationalStudies/types';
+import type { OperationalPointWithTimeAndSpeed } from 'modules/trainschedule/components/DriverTrainScheduleV2/types';
+import type { PathStep } from 'reducers/osrdconf/types';
+
+import useOutputTableData from './hooks/useOutputTableData';
+import TimesStops from './TimesStops';
+import { TableType } from './types';
+
+type TimesStopsOutputProps = {
+  simulatedTrain: SimulationResponseSuccess;
+  pathProperties: PathPropertiesFormatted;
+  operationalPoints: OperationalPointWithTimeAndSpeed[];
+  startTime: string;
+  pathSteps: (PathStep | null)[];
```

Comment:
You can compact the pathSteps in the parent component (null value are only if the path is not fully defined, which is not the case for simulation results). You can also remove the compact line 40.
```suggestion
  pathSteps: PathStep[];
```

Reply:
done